packages/@webex/internal-plugin-user/test/integration/spec/user.js

```diff
@@ -243,138 +386,5 @@ runs.forEach((run) => {
           });
       });
     });
-
-    describe('#get()', () => {
-      it('gets the current user', () =>
```

Comment:
why was the order of the tests been switched?

Reply:
Moved all the API calls with authenticated webex object on top and un-authenticated after that. The mixed order was somewhere messing up the token.

Reply:
sounds like test cleanup is not done properly to me...

Reply:
Might happen due to limitation on number of test users.

Reply:
hmm, i think maybe it's better to investigate that in a different ticket, and if you can revert this change for now would be good

Reply:
Reverted the change